### PR TITLE
Setup hello microverse

### PR DESCRIPTION
**Pull request by abel12-pro**
Hello Code Reviewer, In this project I created the following activities:
-Added the professional readme file
-Added linter for HTML and CSS file
-Added .HTML and .CSS file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello microverse</h1>
```
Comment (mukhammadrizooff) [inline index.html:11]: - [ ] [OPTIONAL] It would be nice if you use `class` attribute for give style 👍🏼

**Review by mukhammadrizooff — APPROVED** [review]
Hi @abel12-pro, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by abel12-pro** [comment]
thanks
*Abel Fikre*
*Work-place :Kotabe  Education University *
*Tel:+251-912-315-562/+251-919-133-790*
*Address: Ethiopia[Addis Ababa ] *

File: index.html (hunk 1 above)
Comment (abel12-pro) [reply index.html:11]: Thanks for your supportive comments.

**Pull request merged**